addons/project/models/project_task.py

```diff
@@ -1163,6 +1163,8 @@ def _change_operator(domain):
         })
         if additional_domain:
             filtered_domain = expression.AND([_change_operator(filtered_domain), additional_domain])
+        else:
+            filtered_domain = _change_operator(filtered_domain)
```

Comment:
Since we're doing it anyway in the `if`, we could just do this:
```suggestion
        filtered_domain = _change_operator(filtered_domain)
        if additional_domain:
            filtered_domain = expression.AND([filtered_domain, additional_domain])
```